bitrise.yml

```diff
@@ -10,6 +10,17 @@ app:
 workflows:
   # ----------------------------------------------------------------
   # --- workflow to Step Test
+  sample:
+    envs:
+    - PROJECT_DIR: "<local_project_dir>"
```

Comment:
This `<local_project_dir>` is just a placeholder and the developer need to replace it, right? If it is then can you add a comment highlighting this? 

(Because I had to look at this too long to realise that this is most probably a placeholder 😂)

Reply:
Updated in https://github.com/bitrise-steplib/steps-project-scanner/pull/148/commits/bdcb0dae4a42cf642571062fc889b7db71efd8b4